i18n/en-US.properties

```diff
@@ -440,6 +440,18 @@ boxui.classification.addClassification = Add Classification
 boxui.classification.classification = Classification
 # Button to edit classification on an item
 boxui.classification.edit = Edit
+# Text to display for users who have not accepted an invitation to collaborate yet
+boxui.collaboratorAvatars.collaboration.pendingCollabText = Pending
+# Label for collaborator avatars
+boxui.collaboratorAvatars.collaboratorAvatarsLabel = Shared with
+# Tooltip text for collaborator expiration badge
+boxui.collaboratorAvatars.collaborators.expirationTooltipText = Access expires on {date}
+# This tooltip indicates that a collaborator is not in the same enterprise of the current user
+boxui.collaboratorAvatars.collaborators.externalCollabTooltipText = {email} is from outside of your company
+# Manage all link text on collaborator list
+boxui.collaboratorAvatars.manageAllLinkText = Manage All
+# Text to display in collaborator list if there are more items
+boxui.collaboratorAvatars.viewAdditionalPeopleText = View additional people
```

Comment:
@jeeyi I changed the ID for these strings because the location of the messages.js file changed. Let me know if that is the preferred pattern.